code/obj/machinery/sleeper.dm

```diff
@@ -484,47 +440,57 @@
 			var/oxy = src.occupant.reagents.get_reagent_amount(src.med_oxy)
 			var/tox = src.occupant.reagents.get_reagent_amount(src.med_tox)
 
+			var/injected_anything = FALSE
+
 			// We always inject this, even when emagged to mask the fact we're malfunctioning.
 			// Otherwise, one glance at the control console would be sufficient.
 			if (rejuv < 10)
 				var/inject_r = 5
-				if ((rejuv + 5) > 10)
+				if ((rejuv + inject_r) > 10)
```

Comment:
Not requiring for this conversion, but if anyone wants to replace all of the magic numbers (`10` in this line, but there's `5` used as well) all over this proc with vars please feel free.